factorialize_a_number.js

```diff
@@ -0,0 +1,9 @@
+function factorialize(num) {
+  if(num === 0 || num === 1) {
+    return 1;
+  } else {
```

Comment:
else nie jest Ci tu potrzebne, dlaczego? bo jezeli wejdzie pierwszy warunek to i tak wyjdziesz z funkcji. else robi Ci tylko niepotrzebne zaglebienie i utrudnia czytanie kodu.